Dockerfile

```diff
@@ -26,7 +26,7 @@ ARG install_dev
 COPY requirements.dev.txt ./
 RUN if [ "${install_dev}" = "y" ]; then pip install -r requirements.dev.txt; fi
 
-COPY --from=client --chown=elife:elife /home/node/client/ ${PROJECT_FOLDER}/client/
+COPY --from=client --chown=elife:elife /home/node/client/dist/ ${PROJECT_FOLDER}/client/dist/
```

Comment:
We only need `dist` when running the server (no need to copy `node_modules` & co)